--- linux-iscsi-4.0.1.1/Makefile.org	2004-03-10 14:12:30.000000000 +0000
+++ linux-iscsi-4.0.1.1/Makefile	2004-03-18 11:40:38.000000000 +0000
@@ -346,7 +346,7 @@
 all:: dirs module daemons utils tools permissions checkkernel checkscsi checkcrypto
 module: dirs checkkernel 
 daemons:  dirs $(OBJDIR)/iscsid
-utils: dirs $(OBJDIR)/iscsi-device $(OBJDIR)/iscsi-iname $(OBJDIR)/iscsi-ls $(OBJDIR)/init $(OBJDIR)/iscsi-id
+utils: dirs $(OBJDIR)/iscsi-device $(OBJDIR)/iscsi-iname $(OBJDIR)/iscsi-ls $(OBJDIR)/init
 #
 test: dirs $(OBJDIR)/bindings
 
