diff -urN linux-iscsi-4.0.2.org/daemon/iscsid.c linux-iscsi-4.0.2/daemon/iscsid.c
--- linux-iscsi-4.0.2.org/daemon/iscsid.c	2005-03-16 05:13:01.000000000 +0100
+++ linux-iscsi-4.0.2/daemon/iscsid.c	2005-03-29 20:03:46.625443328 +0200
@@ -25,7 +25,7 @@
 #include <errno.h>
 #include <poll.h>
 #include <sys/wait.h>
-#include <sysfs/libsysfs.h>
+#include <libsysfs.h>
 
 #include "iscsi-sfnet.h"
 #include "iscsi-protocol.h"
